packages/flutter_tools/bin/xcode_backend.sh

```diff
@@ -130,7 +130,7 @@ BuildApp() {
     --output-file="${derived_dir}/app.flx"                                \
     --snapshot="${build_dir}/snapshot_blob.bin"                           \
     --depfile="${build_dir}/snapshot_blob.bin.d"                          \
-    --working-dir="${build_dir}/flx"                                      \
+    --working-dir="${derived_dir}/flutter_assets"                         \
```

Comment:
Nope, I am however considering renaming the option to `assets-dir` when doing the FLX removal cleanup.

Reply:
Rename sgtm. And the reason we're putting this in `derived_dir` rather than `build_dir` is that we need to reference it from `project.pbxproj` (like the current `app.flx`, `App.framework`, etc.), right?

Reply:
Yes, exactly. Should have added that, sorry.